src/main/kotlin/com/ampnet/reportserviceeth/blockchain/BlockchainEventServiceImpl.kt

```diff
@@ -147,12 +147,12 @@ class BlockchainEventServiceImpl(
             val stableCoinPrecision = getStableCoinPrecision(ContractAddress(asset.issuer), chainProperties)
             events.add(Event(it, chainId.value, log, asset, stableCoinPrecision))
         }
-        skipException { contract.getReleaseEvents(txReceipt) }?.forEach {
-            val log = getLog(logsMap, it)
-            val asset = getAsset(ContractAddress(it.asset), chainProperties)
-            val stableCoinPrecision = getStableCoinPrecision(ContractAddress(asset.issuer), chainProperties)
-            events.add(Event(it, chainId.value, log, asset, stableCoinPrecision))
-        }
+//        skipException { contract.getReleaseEvents(txReceipt) }?.forEach {
```

Comment:
Is this commented out accidentally or on purpose?

Reply:
This is event is not in use anymore, I'll delete it.